examples/simplesamlphp/docker-compose.yml

```diff
@@ -13,10 +13,10 @@ php:
 web:
   image: nginx:alpine
   ports:
-    - "8080:80"
+    - "8081:80"
```

Comment:
@ericmann oops, was from my environment because I use 8080 for lots of things...will revert.